src/components/LinkButton.tsx

```diff
@@ -32,15 +32,9 @@ const LinkButton = (props: Props) => {
     if (onPress) {
       onPress(event)
     } else if (goBack && !to) {
-      // Delay to avoid passing tap to next screen
-      window.setTimeout(() => {
-        history.goBack()
-      }, 100)
+      history.goBack()
     } else if (to && !goBack) {
-      // Delay to avoid passing tap to next screen
-      window.setTimeout(() => {
-        history.push(to)
-      }, 100)
```

Comment:
These timeouts did not seem to be required anymore. I tried it both with a mouse and with a touchscreen and I never saw clicks "bleed through". I removed them because they were making the screen reader's job much harder.

Reply:
Super! There are at least 7 more places we use `window.setTimeout()` to avoid passing taps. Would love to figure out how to remove these as well… as they feel like hacks.